src/backgrounds.rs

```diff
@@ -212,4 +214,27 @@ pub(crate) fn expand_map(
             draw_chunk!(new_chunk, commands, map_atlas_handle);
         }
     }
-}
\ No newline at end of file
+}
+
+macro_rules! set_seed_ {
```

Comment:
Changes to `backgrounds.rs` will probably conflict with #42, should this PR take priority?

Reply:
This change is experimental and minimal, let main take priority.